tachyon/math/matrix/BUILD.bazel

```diff
@@ -48,5 +52,14 @@ tachyon_cc_unittest(
         "//tachyon/base/buffer:vector_buffer",
         "//tachyon/math/finite_fields/test:finite_field_test",
         "//tachyon/math/finite_fields/test:gf7",
-    ],
+    ] + select({
+        "@platforms//cpu:x86_64": [
+            "//tachyon/math/finite_fields/baby_bear:packed_baby_bear_avx2",
+            "//tachyon/math/finite_fields/baby_bear:packed_baby_bear_avx512",
+        ],
+        "@platforms//cpu:aarch64": [
+            "//tachyon/math/finite_fields/baby_bear:packed_baby_bear_neon",
+        ],
+        "//conditions:default": [],
+    }),
```

Comment:
The reason why I do like this is because I wanted to test `avx2` and `avx512` at the same time on x86 if it is built with `avx512`. But I think in this context, just using `PackedBabyBear` seems like to make more sense.